Sources/SpelledPitch.swift

```diff
@@ -1,12 +1,12 @@
 //
-//  Tone.swift
+//  Pitch.swift
```

Comment:
Change to `SpelledPitch.swift` since that's the name of the file.

Reply:
done